package.json

```diff
@@ -41,26 +41,26 @@
     ]
   },
   "dependencies": {
-    "stylelint-config-recommended-scss": "^5.0.0",
-    "stylelint-config-standard": "^23.0.0"
+    "stylelint-config-recommended-scss": "^5.0.1",
+    "stylelint-config-standard": "^24.0.0"
   },
   "devDependencies": {
     "@stylelint/prettier-config": "^2.0.0",
     "@stylelint/remark-preset": "^3.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-stylelint": "^14.0.0",
+    "eslint": "^8.2.0",
+    "eslint-config-stylelint": "^15.0.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-jest": "^25.2.2",
+    "eslint-plugin-jest": "^25.2.4",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-regexp": "^1.5.0",
-    "jest": "^27.2.5",
+    "jest": "^27.3.1",
     "np": "^7.5.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.4.1",
     "remark-cli": "^10.0.0",
-    "stylelint": "^14.0.0"
+    "stylelint": "^14.1.0"
   },
   "peerDependencies": {
-    "stylelint": "^14.0.0"
+    "stylelint": "^14.1.0"
```

Comment:
I don't think removing support for some versions in `peerDependencies` can be done outside of major versions, so this should be left untouched.

Reply:
sure thing, my bad, fixed